roulette/public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
```

Comment:
Starting from the Task 2 the rest of the tasks are done without CRA.
That was the entire purpose of learning React + Webpack configuration from Task 2.
Therefore, consider using your setup from Task 2.

You essentially have 2 options:
- copy all the related React code from `netflix-react/roulette/src` to your `netflix-react/rm2`
- copy all the related project configuration from `netflix-react/rm2/*` to your `netflix-react/roulette/`